Rubberduck.RibbonDispatcher/AbstractRibbonDispatcher/IRibbonDropdown.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace RubberDuck.RibbonDispatcher {
```

Comment:
Wouldn't these interop/wrapper interfaces feel at home in a `Rubberduck.RibbonDispatcher.Controls` namespace?

Reply:
I stuck everything in a single namespace, consistent with Rubberduck's top-level structure, simply as a means of soliciting such comments. I actually have no namespace vision in mind, so that sounds good to me.